### PR TITLE
processhacker-nightly: Add persist

### PR DESCRIPTION
**Pull request by sitiom**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/processhacker-nightly.json [hunk 1]
```diff
@@ -25,6 +25,10 @@
             "Process Hacker"
         ]
     ],
+    "persist": [
+        "ProcessHacker.exe.settings.xml",
```
Comment (rashil2000) [inline bucket/processhacker-nightly.json:29]: What does this file contain? I don't think it has user data.

File: bucket/processhacker-nightly.json [hunk 2]
```diff
@@ -25,6 +25,10 @@
...
+        "usernotesdb.xml"
```
Comment (rashil2000) [inline bucket/processhacker-nightly.json:30]: Would this need to be created if it doesn't exist?

Comment (sitiom) [reply bucket/processhacker-nightly.json:30]: It exists as an empty file in the zip file itself.

File: bucket/processhacker-nightly.json (hunk 1 above)
Comment (sitiom) [reply bucket/processhacker-nightly.json:29]: It contains Process Hacker settings as the filename suggests.

**Pull request merged**